spec/spec.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": 1.0,
+    "ecs": {
+        "version": "1.x"
```

Comment:
should we also include a version field in the log events themselves?

Reply:
I had the same idea, with appenders in mind! Will add that to the spec 💪